### DISCUSSION TITLE
Pls add the Manufacturer „Recycling Fabrik“

### DISCUSSION THREAD

**Discussion by Blue-Code252** [Donkie/Spoolman#527] [Ideas]
Is this alright to add them to the import list of filaments or is the company to small?

I do like the vision of the company.

What are your thoughts on it?

**Comment by Donkie** [comment]
Feel free to add filaments for them yourself, in this repository: https://github.com/Donkie/SpoolmanDB

**Reply by Blue-Code252** [reply]
Thx, didn't know you could do it yourself. XD
